### PR TITLE
Android live ride announement

### LINKED ISSUE #271: Live Ride announcement modal for Android

### PR DESCRIPTION
**Pull request by guytepper**
closes #271 

depends on #256

### CODE REVIEW

File: app/navigators/live-activity-announcement/live-activity-announcement-stack.tsx [hunk 1]
```diff
@@ -28,13 +24,19 @@ export const LiveAnnouncementNavigator = () => (
...
+      //     onPress={() => navigation.navigate("planner")}
+      //     iconStyle={{
+      //       width: 32.5,
+      //       height: 32.5,
+      //       tintColor: Platform.select({ ios: "white", android: "grey" }),
+      //       opacity: 0.5,
+      //       marginTop: 8,
+      //     }}
+      //   />
+      // ),
```
Comment (planecore) [inline app/navigators/live-activity-announcement/live-activity-announcement-stack.tsx:38]: Why is it commented out?

File: app/screens/live-announcement/activity-announcement-screen.android.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+}
+const TEXT: TextStyle = { fontSize: 22, textAlign: "center", lineHeight: 29 }
+
+const LIVE_ACTIVITY_IMAGE: ImageStyle = {
+  width: "100%",
+  height: 355,
+  resizeMode: "contain",
+}
+
+const NOTIFICATION_IMAGE = require("../../../assets/live-ride/-live-ride-notification.png")
```
Comment (planecore) [inline app/screens/live-announcement/activity-announcement-screen.android.tsx:21]: remove leading dash from image file name

File: assets/live-ride/live-ride-intro.png
Comment (planecore) [inline assets/live-ride/live-ride-intro.png:1]: Add English assets

**Review by planecore — CHANGES_REQUESTED** [review]

File: app/navigators/live-activity-announcement/live-activity-announcement-stack.tsx (hunk 1 above)
Comment (guytepper) [reply app/navigators/live-activity-announcement/live-activity-announcement-stack.tsx:38]: I did this to use the arrow icon for Android (close icon feels less suitable there). I'll fix it so it'll appear only for iOS. 
Screenshot 2023-07-22 at 19 02 48

File: assets/live-ride/live-ride-intro-english.png
Comment (planecore) [inline assets/live-ride/live-ride-intro-english.png:1]: Low resolution image, 2 punch holes in the screen. Please frame it the same as [live-ride-intro.png](https://raw.githubusercontent.com/better-rail/app/aaf8fa513e1e3d8e7564949a2f7e9729cf297336/assets/live-ride/live-ride-intro.png)

File: assets/live-ride/live-ride-notification-english.png
Comment (planecore) [inline assets/live-ride/live-ride-notification-english.png:1]: 2 punch holes, some of the status bar is hidden. It's better in 
[live-ride-notification.png](https://raw.githubusercontent.com/better-rail/app/aaf8fa513e1e3d8e7564949a2f7e9729cf297336/assets/live-ride/live-ride-notification.png)

**Pull request merged**